exampleCourse/questions/fibonacciExternal/tests/autograder.py

```diff
@@ -1,80 +1,80 @@
 import py_compile
 
 def main():
-    def test1():
+    def test1(out):
         trueAns = 1
-        studAns = fib.fib(1);
+        studAns = fib.fib(1)
```

Comment:
although this is only an example, this should run in a subprocess (`fib.fib` can print a different result blob to stdout)